spec/models/speech_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe Speech do
```

Comment:
Maybe a good idea to squash this into the commit where you add the implementation of the Content Model? (this is non-blocking, though. I appreciate you might want this here after all the concerns are added)